src/common/errors.mligo

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+#if !COMMON_ERRORS_H
+#define COMMON_ERRORS_H
+
+#include "types.mligo"
+
+let fail_proposal_check (msg : string) : unit =
+  ([%Michelson ({| { FAILWITH } |} : (string * string) -> unit)]
+    ("FAIL_PROPOSAL_CHECK", msg) : unit)
+
+// Xtz transfer amount cannot be 0
+let zero_mutez_err_msg = "ZERO_MUTEZ"
+
+// Xtz transfer amount cannot be smaller than 'min_xtz_amount'
+let too_small_xtz_err_msg = "TOO_SMALL_XTZ_AMOUNT"
+
+// Xtz transfer amount cannot be bigger than 'max_xtz_amount'
+let too_large_xtz_err_msg = "TOO_LARGE_XTZ_AMOUNT"
+
+// Incorrect token amounts locked
+let incorrect_token_amount_err_msg = "INCORRECT_TOKEN_AMOUNT"
```

Comment:
```suggestion
let incorrect_token_amount_err_msg = "WRONG_TOKEN_AMOUNT"
```
P.s. if you make any of these changes remember to update the `registry` and `treasury` `.md` docs